hadoop-cloud-storage-project/hadoop-huaweicloud/pom.xml

```diff
@@ -173,15 +173,27 @@
     </dependency>
     <dependency>
       <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
-      <version>1.7.4</version>
+      <artifactId>powermock-api-mockito2</artifactId>
+      <version>2.0.9</version>
```

Comment:
PowerMock was introduced because Mockito prior to 3.4.0 did not support mocking static method, with the Mockito 4 upgrading, PowerMock can be purged